geonode/static/geonode/js/ol-2.13/lib/deprecated.js

```diff
@@ -25,7 +25,9 @@
  * @requires OpenLayers/Popup/Anchored.js
  * @requires Rico/Corner.js
  */
-
+XMLisOpen = 1;
+XMLhasReceivedHeaders = 2
+XMLisLoaded = 4;
```

Comment:
Inconsistent use of semicolons here. Also could probably declare these as `const`